lib/instances/ImageLabel.lua

```diff
@@ -32,6 +33,12 @@ ImageLabel.properties.Rotation = InstanceProperty.typed("number", {
 	end,
 })
 
+ImageLabel.properties.TileSize = InstanceProperty.typed("UDim2", {
```

Comment:
This looks wrong. There isn't a TileSize property on ImageLabel; there is on ImageButton. Please add to the tests.